test/parallel/test-http-mutable-headers.js

```diff
@@ -103,7 +103,7 @@ function nextTest() {
         break;
 
       case 'writeHead':
-        assert.equal(response.headers['x-foo'], 'bar');
+        assert.equal(response.headers['x-foo'], 'keyboard cat');
```

Comment:
shouldn’t the output here be `keyboard cat, bar`? For incoming http messages node glues the returned header values together with `', '`.